config/config-network.yaml

```diff
@@ -172,3 +172,19 @@ data:
     # fronting Knative with an external loadbalancer that deals with TLS termination and
     # Knative doesn't know about that otherwise.
     default-external-scheme: "http"
+
+    # The CA public certificate used to sign the activator TLS certificate.
+    # It is specified by the secret name, which has the "ca.crt" data field.
+    # Use an empty value to disable the feature (default).
+    #
+    # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
+    #       for now. Use with caution.
+    activator-ca: ""
+
+    # The SAN (Subject Alt Name) used to validate the activator TLS certificate.
+    # It is available only when "activator-ca" is specified.
+    # Use an empty value to disable the feature (default).
+    #
+    # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
+    #       for now. Use with caution.
+    activator-name: ""
```

Comment:
nit: I slightly prefer `activator-san` to `activator-name` as `activator-name` seems too generic. But if you folks have already decided to use `activator-name`, it is OK to me.

Reply:
Yeah, users may misunderstand as `activator-name` means activator's deployment name or something.
Updated the configuration as `activator-san`. Thank you.